src/main/kotlin/org/rust/lang/core/macros/builtin/BuiltinMacroExpander.kt

```diff
@@ -73,9 +71,9 @@ private fun handleFormatMacro(macroName: String, macroText: String, format: RsFo
 
     val namedArguments = mutableSetOf<String>()
     for (argument in format.formatMacroArgList) {
-        val identifier = argument.identifier
-        if (identifier != null) {
-            namedArguments.add(identifier.text)
+        val name = argument.name()
```

Comment:
Hmm, I wanted to write that we should also check here that `argument.eq` isn't `null`, otherwise this could also find things like
`print!("{}", a);` <- here `a` will be returned by `name()`, but it's not a named format argument.

But I checked it and `name()` returns `null` for `a` here, even though it is an identifier, I did not expect that. Well, in that case it's ok I guess :)

Reply:
In the `print!("{}", a);` it's not an identifier but `RsPathExpr`, but I've added an additional check (just in case)